<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createAccount</class>
 <widget class="QWidget" name="createAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>createAccount</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>302</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>24</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:white;</string>
       </property>
       <property name="text">
        <string>Employee Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="emp_name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:white;</string>
       </property>
       <property name="text">
        <string>Employee Id</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="emp_id"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:white;</string>
       </property>
       <property name="text">
        <string>Date Of Birth</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:white;</string>
       </property>
       <property name="text">
        <string>Phone Number</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="emp_number"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font-size:20px;
color:white;</string>
       </property>
       <property name="text">
        <string>Employee Type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="emp_type">
       <item>
        <property name="text">
         <string>Select</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Developer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tester</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Intern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Manager</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="emp_dob"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="ok_btn">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_btn">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>9</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:white;</string>
    </property>
    <property name="text">
     <string>Create Employee</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
